### ISSUE TITLE
Cannot parse CREATE TABLESPACE

### ISSUE THREAD

**Issue by df7cb** [lelit/pglast#98]
pglast 3.8 on Debian's Python 3.9.10:

```
>>> pglast.parse_sql("create tablespace foo location '/bar'")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "pglast/parser.pyx", line 268, in pglast.parser.parse_sql
  File "pglast/ast.pyx", line 5866, in pglast.parser.create
  File "pglast/ast.pyx", line 1506, in pglast.parser.create_RawStmt
  File "pglast/ast.pyx", line 6000, in pglast.parser.create
  File "pglast/ast.pyx", line 2373, in pglast.parser.create_CreateTableSpaceStmt
  File "pglast/parser.pyx", line 226, in pglast.parser.Displacements.__call__
TypeError: '<=' not supported between instances of 'int' and 'bytes'
```

**Comment by lelit** [comment]
Thanks, I will try to understand and cure the problem in the following days.

**Comment by lelit** [comment]
Should be fixed in v3.

**Comment by df7cb** [comment]
Thanks, works now!

**Comment by lelit** [comment]
Released in v3.9.
